scripts/test/shared.py

```diff
@@ -57,6 +57,11 @@ def parse_args(args):
         '--binaryen-root', dest='binaryen_root', default='',
         help=('Specifies a path to the root of the Binaryen repository tree.'
               ' Default: the directory where this file check.py resides.'))
+    parser.add_argument(
+        '--binaryen-test-out', dest='binaryen_test_out', default='',
```

Comment:
I think simply calling this `--out-dir` would be more clear and concise.    The binaryen-test part is kind of redundant